### DISCUSSION TITLE
Allow the use of different "Heading" for different event colors in the daily note

### DISCUSSION THREAD

**Discussion by Drakon5999** [davish/obsidian-full-calendar#416] [Feature Requests]
The division of events by color is a very convenient feature. 
However, it doesn't quite work when storing events in the daily note. 
It would be very cool to be able to store events of different colors in the daily note, for example under different Heading

**Comment by davish** [comment]
@Drakon5999 thanks for the feature request!

**Comment by felipejoribeiro** [comment]
Different headers would be interesting, but maybe something more basic in the getgo would be adding a new tag to the list with a HEX color ou RGB value, something like:

```markdown
played elden ring [startTime:: 11:20]  [endTime:: 12:30] [taskColor:: #1E6832]
```

And allow the user to change the color from the UI

**Comment by YouFoundJK** [comment]
Check out the [advanced categorization feature](https://youfoundjk.github.io/plugin-full-calendar/events/categories/) in my fork [here](https://github.com/YouFoundJK/plugin-full-calendar/releases/latest).
